src/IotCentral/IotCentral/IotCentral/SetAzureRmIotCentralApp.cs

```diff
@@ -46,6 +46,14 @@ public class SetAzureRmIotCentralApp : IotCentralFullParameterSetCmdlet
         [ValidateNotNullOrEmpty]
         public Hashtable Tag { get; set; }
 
+        [Parameter(
```

Comment:
these are generated by autotest correct?

Reply:
Not that I'm aware of. I don't THINK powershell uses autorest, only arm swagger file. Did I miss something?

Reply:
Just checked. Yea, I think you are right. We need to do it ourselves!

Reply:
We are using autorest to generate a few modules such as "Az.Databricks", but most of them are still hand written.